aio/aio-proxy/aio_proxy/response/helpers.py

```diff
@@ -1,5 +1,6 @@
 import json
```

Comment:
note for future : 
This file actually mix very different stuf and it would be interesting to split helpers by topic :

- is_dev_env is related to infra / env, how the app operate
- serialize_error_text and hash_string are string helpers
- get_value is also a basic helper
- format_nom_complet actually bears quite an important business logic and should be placed with other business logic heavy code/helpers

Reply:
I prefer to do it in the upcoming refactoring PRs.